servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultMultiAddressUrlHttpClientBuilder.java

```diff
@@ -453,4 +463,29 @@ public MultiAddressHttpClientBuilder<HostAndPort, InetSocketAddress> followRedir
         this.redirectConfig = config;
         return this;
     }
+
+    @Override
+    public MultiAddressHttpClientBuilder<HostAndPort, InetSocketAddress> defaultHttpPort(final int port) {
+        this.defaultHttpPort = verifyPortRange(port);
+        return this;
+    }
+
+    @Override
+    public MultiAddressHttpClientBuilder<HostAndPort, InetSocketAddress> defaultHttpsPort(final int port) {
+        this.defaultHttpsPort = verifyPortRange(port);
+        return this;
+    }
+
+    /**
+     * Verifies that the given port number is within the allowed range.
+     *
+     * @param port the port number to verify.
+     * @return the port number if in range.
+     */
+    private static int verifyPortRange(final int port) {
+        if (port < 0 || port > 0xFFFF) {
+            throw new IllegalArgumentException("Provided port number out of range: " + port);
```

Comment:
1. Missed "is" before "out"?
2. For consistency with any other validation we have, consider specifying expected range in the message.